content/learn/accessibility/keyboard-testing/index.md

```diff
@@ -8,13 +8,13 @@ showHeadings: false
 showInNav: true
 ---
 
-# Keyboard testing procedures for accessibility
+# Keyboard & Manual testing procedures for accessibility
 
 ### 1. Use the Tab key to move through a page or component
 
 Ensure that navigation with the Tab key meets the following criteria:
 
-1. The Tab key places focus on all focusable elements.
+1. The Tab key places focus on all [focusable elements](https://allyjs.io/data-tables/focusable.html) (e.g., form controls, navigation elements).
```

Comment:
I'd change "(e.g., form controls, navigation elements)" to "such as form controls and navigation elements."